tests/system/test_backup_api.py

```diff
@@ -93,11 +93,11 @@ def database_version_time(shared_database):
 
 
 @pytest.fixture(scope="session")
-def second_database(shared_instance, database_operation_timeout):
+def second_database(shared_instance, database_operation_timeout, database_dialect):
     database_name = _helpers.unique_id("test_database2")
     pool = spanner_v1.BurstyPool(labels={"testcase": "database_api"})
     database = shared_instance.database(
-        database_name, ddl_statements=_helpers.DDL_STATEMENTS, pool=pool
+        database_name, ddl_statements=_helpers.DDL_STATEMENTS, pool=pool, database_dialect=database_dialect
```

Comment:
For PG, first you create a database and then apply update ddl. It doesn't allow passing ddl statements at creation time. This code should most likely fail